.github/renovate.json5

```diff
@@ -8,7 +8,33 @@
   ignorePaths: [
     "**/tests/**",
   ],
+  regexManagers: [
+    {
+      customType: 'regex',
+      fileMatch: [
+        '^Cargo.toml$',
+      ],
+      matchStrings: [
+        'rust-version.*?(?<currentValue>\\d+\\.\\d+(\\.\\d+)?)',
+      ],
+      depNameTemplate: 'latest-msrv',
+      packageNameTemplate: 'rust-lang/rust',
+      datasourceTemplate: 'github-releases',
```

Comment:
Just mind that `1.71.1` is not yet in [GitHub Release](https://github.com/rust-lang/rust/releases).